repos/keg-components/src/utils/helpers/handleRefUpdate.js

```diff
@@ -8,6 +8,9 @@ import { isObj, checkCall } from '@keg-hub/jsutils'
  * @returns {Object} - Combined child components
  */
 export const handleRefUpdate = (ref, update) => {
+  if (!ref) return update
```

Comment:
These 2 lines, could just be this => `if(!ref || !update) return ref || update`
Although, I'm not sure the code is correct.
* If `ref` is a valid ref, and `update` is `false`, It should still update the `ref.current` with `false`
* But with this line `if (!update) return ref`, that never happens

So I think something like this would be better =>
```js
export const handleRefUpdate = (ref, update) => {
  // Update the ref based on it's type
  return !ref
    ? update
    ? isObj(ref) && 'current' in ref
      ? (ref.current = update)
      : checkCall(ref, update)
}
```

Might need some testing to ensure I didn't miss something.

Reply:
tested and updated 👍